bin/ch-run.c

```diff
@@ -73,6 +74,7 @@ const struct argp_option options[] = {
    { "unset-env",      -7, "GLOB", 0, "unset environment variable(s)" },
    { "verbose",       'v', 0,      0, "be more verbose (can be repeated)" },
    { "version",       'V', 0,      0, "print version and exit" },
+   { "warnings",      -16, "NUM",  0, "log NUM warnings and exit" },
```

Comment:
Is it possibly to bury this under some debug option? Or maybe we should worry about that when we get more similar stuff?

Reply:
I'm leaning towards waiting until we have more things to group under such an option, but I'd be willing to discuss further.